### ISSUE TITLE
External speaker support for `AN/PRC-117F`

### ISSUE THREAD

**Issue by rautamiekka** [IDI-Systems/acre2#1207]
**Is your feature request related to a problem?**
Not really, although you could argue that long-range radio trading an internal speaker found in 152 for a quad range creates a communication problem in the field and/or at HQ since only you can hear that radio.

**Solution you'd like:**
There was a mention in https://github.com/IDI-Systems/acre2/issues/555 of such a feature having been internally implemented by some group as a component, so maybe my description below is overcomplicating it.

My 2 possible ways I imagined this being implemented:
1. Add a separate misc item into arsenal, something that denotes it's just a separate module, like `{RADIO NAME} speaker module`.
1.1. The ACE interact menu for every carried 117 would have an option to connect a speaker if a spare one is carried, thus allowing to easily choose which 117 has a speaker. The same way double-clicking a 117 in the inventory would show the options, thus allowing to skip using ACE interact on a per-117 basis.
1.2. The speaker would ideally not show up in inventory, instead the radio one should mention the speaker, but if the speaker is its own actual item, the world graphics could just reuse a big pouch from the base game.
2. Add a separate 117 into arsenal, named something like `{RADIO NAME} (with speaker)`.

In both cases:
- The radio would have the same 'toggle speaker' option as 152 (as mentioned above, module would have to be connected 1st), and functions exactly the same.
- The part of the radio's status screen would say a speaker is connected, whatever the actual wording would be in such case in real life.
- The graphics (the radio's `Open menu` in particular) don't necessarily need to change nor new ones be added.
- The speaker appropriately adds some weight. I couldn't find anything regarding just how much do they weigh, I imagine max 2kg (4.4lbs).
- Assign a variable to the radio with these possible values (I imagine this would be more optimal from coding standpoint since the radio wouldn't require its own class name):
  - `0` = no speaker connected. This the default, at least if the speaker is a separate item.
  - `1` = speaker connected but disabled. Functions exactly like one wasn't connected, but can be turned on/off anytime from ACE interact.
  - `2` = speaker connected and enabled. Functions exactly like on a 152.

**Alternatives you've considered:**
Couldn't think of any; likely beyond my coding level to implement, too.

**Additional context:**
The 117 does have multiple options for an external speaker in real life, but I could find just 1 that's powered by the radio itself, specifically the `10570-3500-22` in [multiband-multimission-radio.pdf](https://github.com/IDI-Systems/acre2/files/8810078/multiband-multimission-radio.pdf) from https://pdf4pro.com/cdn/multiband-multimission-radio-661dc.pdf

**Comment by WelshBanana** [comment]
I disagree with the speaker being for a manpack variant however I could see its uses as a vehicle mounted variant.

**Comment by rautamiekka** [comment]
Better leave that to the Players to decide.
